### DISCUSSION TITLE
Ncurses installer?

### DISCUSSION THREAD

**Discussion by bolt926** [caernarferon/caveman-linux#50] [Ideas]
Making an installer with a GUI will probably be easier for newbies.

**Comment by polandlover123** [comment]
Nooo tui bad! No ncurses!

**Reply by bolt926** [reply]
Why is TUI bad?

**Comment by spacefault** [comment]
I think an ncurses installer would be good for a potential installer. Maybe installer-v2.5?
For now we should consider finishing installer-v2 first and then see what we can do from there.

**Reply by polandlover123** [reply]
Ncurses 😡
